<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WatermarkDialog</class>
 <widget class="QDialog" name="WatermarkDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>235</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>请输入水印内容</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../resource/picture/watermark.png</normaloff>../resource/picture/watermark.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>水印内容：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="watermarkWord"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>透明度：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="alpha">
         <property name="text">
          <string>0.5</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>倾斜角度：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ankle">
         <property name="text">
          <string>20</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>颜色：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="colorComboBox">
         <item>
          <property name="text">
           <string>红色</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../resource/picture/红色.png</normaloff>../resource/picture/红色.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>蓝色</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../resource/picture/蓝色.png</normaloff>../resource/picture/蓝色.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>绿色</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../resource/picture/绿色.png</normaloff>../resource/picture/绿色.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>灰色</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../resource/picture/灰色.png</normaloff>../resource/picture/灰色.png</iconset>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="confirmButton">
         <property name="text">
          <string>确定</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="cancleButton">
         <property name="text">
          <string>取消</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
